apps/src/templates/sectionAssessments/sectionAssessmentsRedux.js

```diff
@@ -367,6 +367,15 @@ export const getMultipleChoiceSurveyResults = (state) => {
   });
 };
 
+// Returns a boolean.  The selector function checks if the current assessment Id
+// is in the surveys structure.
+export const currentAssessmentIsSurvey = (state) => {
```

Comment:
nit: the convention i usually go with for methods that return a boolean is for the method name to start with `is`, so this would be `isCurrentAssessmentSurvey`

Reply:
Fixed!